index.php

```diff
@@ -10,18 +10,49 @@ function e($string)
     return htmlspecialchars($string);
 }
 
+/**
+ * @param $amount
+ * @return string
+ */
 function format_amount($amount)
```

Comment:
please do typing for input params and add type for output results. PhpDoc is good to have and it was required up to php 7. Since php 7 it's possible to do typing at language level.

So it would be something like `function format_amount(int $amount): string`

Reply:
не, я не про phpdoc, а про саму нотацию функции, смотри мой пример

Reply:
9.8 => 10,
ceil - ceiling -> вверх

9.8 => 9
floor(9.8)

9.8 => 10
9.4 => 9
round(9)

Reply:
PHP
float - плавающая точка, 9.456(75457657567567)
int - целое
---
Java
double - 9.45675457657567567